IFC4x3/PropertyConstants/t/THOUGHBOLTED_37J6KviK95fh08mX_vHy5u.xml

```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="THOUGHBOLTED_37J6KviK95fh08mX_vHy5u" Name="THOUGHBOLTED" UniqueId="c74c6539-b142-45a6-b008-c21fb947c178" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="THOUGHBOLTED_37J6KviK95fh08mX_vHy5u" Name="THOUGHBOLTED" UniqueId="c74c6539-b142-45a6-b008-c21fb947c178">
+	<Documentation>mooring device is anchored via tighted bolts through the device and substrate</Documentation>
```

Comment:
Capitalize 'M' in 'mooring'. 'THOUGHBOLTED' typo? Should be 'THROUGHBOLTED'?

Reply:
ha! i was definently half asleep when i did this commit!!